src/main/java/com/google/cloud/spanner/jdbc/JdbcParameterStore.java

```diff
@@ -775,6 +808,13 @@ private Builder setArrayValue(ValueBinder<Builder> binder, int type, Object valu
         case Types.LONGVARBINARY:
         case Types.BLOB:
           return binder.toBytesArray(null);
+        case ProtoMessageType.VENDOR_TYPE_NUMBER:
+        case ProtoEnumType.VENDOR_TYPE_NUMBER:
+          return binder.to(
```

Comment:
Makes sense, Knut. However, I wonder should we add this change in [PR#1224](https://github.com/googleapis/java-spanner-jdbc/pull/1224) so that all changes for supporting untyped null gets maintained at one place?
Please let me know your views on this.

Reply:
Ah, yeah, good point.